.copywrite.hcl

```diff
@@ -0,0 +1,31 @@
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2020
+
+  header_ignore = [
+    # changie tooling configuration and CHANGELOG entries (prose)
+    ".changes/unreleased/*.yaml",
+    ".changie.yaml",
+
+    # GitHub issue template configuration
+    ".github/ISSUE_TEMPLATE/*.yml",
+
+    # GitHub Actions workflow-specific configurations
+    ".github/labeler-*.yml",
+
+    # golangci-lint tooling configuration
+    ".golangci.yml",
+
+    # GoReleaser tooling configuration
+    ".goreleaser.yml",
+
+    # Release Engineering tooling configuration
+    ".release/*.hcl",
+
+    # Proto and protoc generated files
```

Comment:
Along with the auto-generated files, I also ignored the `.proto` file. I'm assuming that is okay? I believe comments would persist to the auto-generated file as well if we wanted to just ignore the auto-generated files only, i.e. `*.pb.go`